### PR TITLE
Add auth section for each external service page

### PR DESCRIPTION
**Pull request by josunect**
To complete the external services information I've included an auth section for each page: 

- Grafana https://deploy-preview-814--kiali.netlify.app/docs/configuration/p8s-jaeger-grafana/grafana/
- Prometheus https://deploy-preview-814--kiali.netlify.app/docs/configuration/p8s-jaeger-grafana/prometheus/ 
- Tempo https://deploy-preview-814--kiali.netlify.app/docs/configuration/p8s-jaeger-grafana/tracing/tempo/
- Jaeger https://deploy-preview-814--kiali.netlify.app/docs/configuration/p8s-jaeger-grafana/tracing/jaeger/

### CODE REVIEW

File: content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md [hunk 1]
```diff
@@ -51,3 +51,22 @@ spec:
 The described configuration is done in the Kiali CR when Kiali is installed using the Kiali Operator. If Kiali is installed with the Helm chart then the correct way to configure this is via regular --set flags.
 {{% /alert %}}
 
+### Grafana authentication configuration
+
+The Kiali CR provides authentication configuration that will be used also for querying the version check to provide information in the Mesh graph.
```
Comment (nrfox) [inline content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md]:
```suggestion
The Kiali CR provides authentication configuration that will be used to connect to your grafana instance and for detecting your grafana version in the Mesh graph. 
```

File: content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md [hunk 2]
```diff
@@ -51,3 +51,22 @@ spec:
...
+    grafana:
+      enabled: true
+      auth:
+        ca_file: ""
+        insecure_skip_verify: false
+        password: "pwd"
+        token: ""
+        type: "basic"
+        use_kiali_token: false
+        username: "user"
```
Comment (nrfox) [inline content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md:76]: Can you also provide an example of using a `secret` or at least link to it here? The format of this almost always trips people up but it's usually how people want to provide credentials.

Comment (josunect) [reply content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md:76]: Yes, I think that will be good idea. Maybe adding a FAQ entry and link it from the rest of the pages, so the information is not duplicated

Comment (josunect) [reply content/en/docs/Configuration/p8s-jaeger-grafana/grafana.md:76]: @nrfox added this https://deploy-preview-814--kiali.netlify.app/docs/faq/authentication/#how-to-configure-a-secret-as-password-for-external-services based in @jmazzitelli instructions (Mazz you might want to have a look?) I've tested it in minikube and works as expected.

**Review by nrfox — APPROVED** [review]

**Pull request merged**